genome_kit/variant_genome.py

```diff
@@ -198,10 +199,29 @@ def find_motif(self, interval, motif, match_position=0, find_overlapping_motifs=
     def __repr__(self):
         return '<VariantGenome {} with {} variants>'.format(self.reference_genome, len(self.variants))
 
+    @property
+    def reference_genome(self):
+        return self.genome.reference_genome
+
+    def __eq__(self, other):
+        if not isinstance(other, VariantGenome):
+            return False
+        if self.genome != other.genome:
+            return False
+        if self.variants != other.variants:
+            return False
+        return True
+
     def __getattr__(self, name):
         # Intercept any attribute requests that weren't found on the VariantGenome object itself,
         # and forward those requests to the Genome object we're wrapping.
 
-        # TODO: This is problematic: See DGENGINE-1401
-
         return self.genome.__getattribute__(name)
+
+    def __getstate__(self) -> bytes:
+        return pickle.dumps([self.genome, self.variants])
```

Comment:
I know maybe this isn't consistent, but do you want to set a pickle version?  otherwise this will be dependent on the python runtime